OpenTrace.xcodeproj/xcshareddata/xcschemes/opentrace-production.xcscheme

```diff
@@ -31,7 +31,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
this is so we always run Debug builds from Xcode.

command line builds (fastlane for AppCenter) will still use Release.